tests/test_example.py

```diff
@@ -2,7 +2,7 @@
 
 
 def test_google(py: Pylenium):
-    py.visit('https://google.com')
-    py.get('[name="q"]').type('puppies')
-    py.get('[name="btnK"]').submit()
-    assert py.should().contain_title('puppies')
+    py.visit('https://lambdatest.github.io/sample-todo-app/')
```

Comment:
this is not the old url, right?

Reply:
Old URL was https://todo-app.lambdatest.io/index.html

Reply:
cool 👍